server/services/interventionsService.ts

```diff
@@ -399,4 +405,41 @@ export default class InterventionsService {
       headers: { Accept: 'application/json' },
     })) as SubmittedActionPlan
   }
+
+  async getActionPlanAppointments(token: string, actionPlanId: string): Promise<ActionPlanAppointment[]> {
+    const restClient = this.createRestClient(token)
+    return (await restClient.get({
+      path: `/action-plan/${actionPlanId}/appointments`,
+      headers: { Accept: 'application/json' },
+    })) as ActionPlanAppointment[]
+  }
+
+  async createActionPlanAppointment(
+    token: string,
+    actionPlanId: string,
+    appointment: ActionPlanAppointment
+  ): Promise<ActionPlanAppointment> {
+    const restClient = this.createRestClient(token)
+    return (await restClient.post({
+      path: `/action-plan/${actionPlanId}/appointment`,
+      headers: { Accept: 'application/json' },
+      data: { ...appointment },
+    })) as ActionPlanAppointment
+  }
+
+  async updateActionPlanAppointment(
+    token: string,
+    actionPlanId: string,
+    appointment: ActionPlanAppointment
```

Comment:
ActionPlanAppointment contains a sessionNumber. In the case of an update is it applicable?

Reply:
I do favour it as a parameter as it is part of the key to identifying an appointment and thus something that can't change, whereas the body contains elements that can change.

Reply:
done